client/components/Mentions.vue

```diff
@@ -211,14 +211,20 @@ export default {
 				this.isOpen = false;
 			}
 		},
-		openPopup() {
+		togglePopup() {
 			this.isOpen = !this.isOpen;
 
 			if (this.isOpen) {
 				this.isLoading = true;
 				socket.emit("mentions:get");
+				eventbus.on("escapekey", this.closePopup);
```

Comment:
Since `closePopup` is called when escape is hit, but it doesn't unhook the event bus, we can end up adding a bunch of callbacks. Ex:

1. `togglePopup` is called. Callback is added.
2. Escape is pressed. Popup is  closed.
3. `togglePopup` is called. Callback is added.

We now have 2 callbacks added. And we could continue doing this to get as many callbacks as we'd like.

It might be simplest to add the hook when the component is mounted and then remove it when the component is destroyed. This'd take care of my other comment as well.